Add spring 2019 MLADS walk-through.

yangsiyu007 opened the pull request.

jennifermarsman commented:
In section 6, let's explicitly tell them to use a Linux VM too...

jennifermarsman commented:
NOTE: when doing docker pull on the ACR, I had issues until the ACR name was in all lower case (must be doing a case-sensitive lookup somewhere).  I was able to create it and docker push it just fine with mixed casing, but needed lower case to docker pull from the VM in section 6.

yangsiyu007 commented:
Thanks again @jennifermarsman for testing it on Windows! I've incorporated your suggestions and added a section on "Deploy your own model" (also pointing to example model files in our Exampels folder).
